common/djangoapps/track/views/tests/test_views.py

```diff
@@ -1,22 +1,27 @@
 # pylint: disable=missing-docstring,maybe-no-member
 
-from track import views
-from track.middleware import TrackMiddleware
 from mock import patch, sentinel
-from freezegun import freeze_time
```

Comment:
I don't know this library, but I'm wondering if it's used anywhere else. (And if not, should we remove it from requirements.txt?)